app.rb

```diff
@@ -0,0 +1,135 @@
+require 'json'
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book_manager'
+require_relative 'person_manager'
+require_relative 'rental_manager'
+
+class App
+  def initialize(book_manager, person_manager, rental_manager)
+    @book_manager = book_manager
+    @person_manager = person_manager
+    @rental_manager = rental_manager
+    load_data
+  end
+
+  def list_books
+    puts 'List of Books:'
+    @book_manager.books.each do |book|
+      puts "#{book.title} by #{book.author}"
+    end
+  end
+
+  def list_people
+    puts 'List of People:'
+    @person_manager.people.each do |person|
+      puts "#{person.name} (ID: #{person.id}, Class: #{person.class})"
+    end
+  end
+
+  def create_person
+    puts 'Enter person details:'
+    name = get_user_input('Name: ')
+
+    puts 'Select person type:'
+    puts '1. Student'
+    puts '2. Teacher'
+    choice = get_user_input('Enter the option (1/2): ').to_i
+
+    case choice
+    when 1
+      age = get_user_input('Age: ').to_i
+      parent_permission = get_user_input('Has parent permission? (y/n): ').casecmp('y').zero?
+      @person_manager.create_student(name, age: age, parent_permission: parent_permission, classroom: 'Some Classroom')
+      puts "#{name} created as a Student."
```

Comment:
- [ ] Good job so far, :clap:, However, your app throws errors when I type `y` for parent permission and thus unable to create person.json data. Please resolve it.
- Below is the screenshot, please pay attention on the red arrow. :arrow_down: 
![Error](https://github.com/faisalakb/Ruby_OOP_school_library/assets/43528594/3c04ab4d-9633-4cb8-b9d2-3b1cddf48c67)